ushahidi/platform-client
# removed private toggle button from survey-edit window fixes #3909

This pull request makes the following changes:
- Removes private toggle button from the survey-edit modal window.
- Private toggle button is removed for Title and description fields only.

Testing checklist:
- Go to settings
- Go to survey and select any survey to edit.
- Click on its title/description option.
- [ ] **make private** toggle button should not be displayed.

Fixes ushahidi/platform#3909 .

Ping @ushahidi/platform
